plg_packages/plg_dbs/setup.py

```diff
@@ -0,0 +1,25 @@
+from setuptools import setup, find_packages
+
+VERSION = '0.0.1' 
+DESCRIPTION = 'operations for playground databases'
+
+setup(
+        name="plg_dbs",
```

Comment:
After discussing with Yonatan,  we agree that this package should not called plg but rather recipe_grabber or something similar.

Reply:
So changing plg everywhere to recipe_grabber?
I will see if I can find some package naming and structure conventions somewhere.